CHANGELOG.md

```diff
@@ -92,6 +92,9 @@
 * Updated to CocoaPods 1.9.0.  
   [David Jennes](https://github.com/djbe)
   [#619](https://github.com/SwiftGen/SwiftGen/pull/619)
+* Updated Pods and Gems dependencies
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SwiftGen/SwiftGen/blob/4900d105ba6e710fbc57d0ee405a9ea137dd73f1/CHANGELOG.md#L95"></span>Line describing your entry should end with a period and 2 spaces.</td>
    </tr>
  </tbody>
</table>